grammar.js

```diff
@@ -67,7 +68,34 @@ module.exports = grammar({
   word: $ => $.identifier,
 
   rules: {
-    translation_unit: $ => repeat($._top_level_item),
+    translation_unit: $ => repeat(choice(
```

Comment:
It's not clear what is the difference with top_level_item.
I'd rather make the difference explicit by introducing intermediate rules like
top_level_item_no_xxx
statement_no_xxx

Reply:
Like rename `_top_level_item` to `_block_item` and wrap the above to the actual `_top_level_item` and inline both?

Reply:
What do you mean by `no_xxx`? How do you want to count something?

Reply:
I mean that the list looks like the inline content of the top_level_item and the statement rule, but obviously it's not, so there must be certain statements that are not in it (it's still not clear to me which statements are not in it), which is why I proposed to rename to statements_no_xxx to be more explicit about the difference.

Reply:
binary_expression, since that conflicts w/ pointer declarators, I'll rework it and add a comment explaining which one's missing